### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Travis-55** [ricardovelero/ai-saas#1]
Hey Ricardo where did u download the logo from (nice project btw)

**Comment by ricardovelero** [comment]
Thanks! That logo is from my wife’s travel agency and was designed by a graphic designer. Her business folded, victim of the pandemic. So she donated those graphic assets to me so I can build something with it.

**Comment by Travis-55** [comment]
do u have any sites that provide high quality logos?

**Comment by ricardovelero** [comment]
No, I don't, but I would love to have one good source of cool logos too.
